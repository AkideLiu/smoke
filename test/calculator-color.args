--color fixtures/calculator fixtures/calculator.rb
