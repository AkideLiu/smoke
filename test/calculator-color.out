addition
[32m  succeeded[0m
complex
[31m  input:            (8 + 2i) + (7 - 4i)
  actual output:    2
  expected output:  15 - 2i[0m
division
[32m  succeeded[0m
infix-words-error
[31m  input:            3 to-the-power-of 3
  actual error:     fixtures/calculator.rb:4:in `<main>': undefined method `to-the-power-of' for 3:Fixnum (NoMethodError)
  expected error:   ./fixtures/calculator.rb:3:in `<main>': undefined method `to-the-power-of' for 3:Fixnum (NoMethodError)[0m
multiple-digits
[32m  succeeded[0m
multiplication
[32m  succeeded[0m
parens
[31m  input:            (6 + 3) * 2
  actual output:    3
  expected output:  18[0m
subtraction
[32m  succeeded[0m
three-numbers
[31m  input:            5 + 7 + 12
  actual output:    12
  expected output:  24[0m

[31m9 tests, 4 failures[0m
